### PR TITLE
non-comp SC and HLR create DecisionReviewTask on complete!

### PR DESCRIPTION
**Pull request by pkarman**
connects #7990 

### Description
Create DecisionReviewTask when intake completes for non-comp HLR and SC.

### CODE REVIEW

File: app/models/claim_review.rb [hunk 1]
```diff
@@ -48,6 +48,12 @@ def create_issues!(new_issues)
     end
   end
 
+  def create_non_comp_task!
+    return if tasks.any? { |task| task.is_a?(DecisionReviewTask) } # TODO: more specific check?
+    # TODO: better user?
```
Comment (pkarman) [inline app/models/claim_review.rb]: not sure who should own this task, or if it matters at this point.

File: app/models/claim_review.rb [hunk 2]
```diff
@@ -48,6 +48,12 @@ def create_issues!(new_issues)
     end
   end
 
+  def create_non_comp_task!
+    return if tasks.any? { |task| task.is_a?(DecisionReviewTask) } # TODO: more specific check?
```
Comment (leikkisa) [inline app/models/claim_review.rb]: This is just in case someone completes a claim review twice?

File: app/models/claim_review.rb (hunk 1 above)
Comment (leikkisa) [reply app/models/claim_review.rb]: Makes sense. Do we even need to assign it to anyone yet?

Comment (pkarman) [reply app/models/claim_review.rb]: apparently `assigned_to` is required to pass validation.

**Pull request merged**